routes/issues.js

```diff
@@ -0,0 +1,8 @@
+const express = require("express");
+const router = express.Router();
+const issues = require("../controllers/issues");
+
+router.get("/:repo", issues.getIssues);
+router.post("/updates", issues.issueUpdates);
```

Comment:
There is no validation required here? Aren't we sending any body?

Reply:
I can not see the task validator being used here. Am I looking at it the wrong way?

Reply:
This route is used for recieving updates from the GitHub webhook whenever there are any changes done on an issue. Hence I didn't add any validation here. Also this is for the issues route and not tasks -- hence haven't added task validator as well.